go.mod

```diff
@@ -31,18 +31,18 @@ require (
 	go.opentelemetry.io/otel/sdk/metric v1.25.0
 	go.uber.org/zap v1.27.0
 	golang.org/x/crypto v0.22.0
-	golang.org/x/oauth2 v0.16.0
```

Comment:
I would prefer library upgrades to be in a separate commit -  or are these upgrades mandatory for the cloud run code in this PR?

Reply:
These upgrades came after importing the run api.